eng/pipelines/handlers.yml

```diff
@@ -122,42 +122,40 @@ stages:
           steps:
             - template: common/build-windows.yml
 
-  # - stage: windows_cake_validation
-  #   displayName: Windows Cake Validation
-  #   dependsOn: []
-  #   jobs:
-  #     - ${{ each BuildCondition in parameters.BuildConfigurations }}:
-  #       - job: win_hosted_${{ BuildCondition }}
-  #         workspace:
-  #           clean: all
-  #         displayName: Build Windows Phase (${{ BuildCondition }})
-  #         condition: or(
-  #           ${{ parameters.BuildEverything }},
-  #           ne(variables['Build.Reason'], 'PullRequest'),
-  #           eq('${{ BuildCondition }}', 'Release'))
-  #         timeoutInMinutes: 60
-  #         pool:
-  #           name: $(vs2019VmPool)
-  #           vmImage: $(vs2019VmImage)
-  #           demands: [ msbuild ]
-  #         steps:
-  #           - powershell: |
-  #                 & dotnet cake --target=dotnet
-  #                 $env:PATH = (Join-Path '$(System.DefaultWorkingDirectory)' 'bin/dotnet') + [IO.Path]::PathSeparator + $env:PATH
-  #                 & dotnet cake --target=VS-WINUI-CI --configuration=${{ BuildCondition }}
-  #           - task: CopyFiles@2
-  #             displayName: 'Copy Bin Log Files'
-  #             condition: always()
-  #             inputs:
-  #               Contents: |
-  #                 **/*.binlog
-  #               TargetFolder: $(build.artifactstagingdirectory)
-  #               flattenFolders: true
-  #           - task: PublishBuildArtifacts@1
-  #             condition: always()
-  #             displayName: publish artifacts
-  #             inputs:
-  #               ArtifactName: win_build_cake
+  - stage: windows_cake_validation
+    displayName: Windows Cake Validation
+    dependsOn: []
+    jobs:
+      - ${{ each BuildCondition in parameters.BuildConfigurations }}:
+        - job: win_hosted_${{ BuildCondition }}
+          workspace:
+            clean: all
+          displayName: Build Windows Phase (${{ BuildCondition }})
+          condition: or(
+            ${{ parameters.BuildEverything }},
+            ne(variables['Build.Reason'], 'PullRequest'),
+            eq('${{ BuildCondition }}', 'Release'))
+          timeoutInMinutes: 60
+          pool:
+            name: $(vs2019VmPool)
+            vmImage: $(vs2019VmImage)
+            demands: [ msbuild ]
+          steps:
+            - powershell: |
```

Comment:
I think something needs to provision VS 16.10, using this file for now would work: https://github.com/dotnet/maui/blob/main/eng/provisioning/vs2019.csx